src/SortableContainer/index.js

```diff
@@ -339,12 +341,21 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 				if (transitionDuration) {
 					node.style[`${vendorPrefix}TransitionDuration`] = `${transitionDuration}ms`;
 				}
+
+				const margin = getElementMargin(node);
+				// margins merge so we subtract them
+				const delta = (
+					axis === 'x'
+					? this.width  + this.margin.left + this.margin.right  - margin.left
+					: this.height + this.margin.top  + this.margin.bottom - margin.top
+				);
+
```

Comment:
@clauderic 
I found a problem with this de-merging of margins, when `margin.left > this.margin.left` it doesn't work.
There are other cases where all the items below need re-positioning.
So please don't merge yet